presto-main/src/main/java/com/facebook/presto/operator/aggregation/state/StateCompiler.java

```diff
@@ -536,9 +563,9 @@ private static FieldDefinition generateGroupedField(ClassDefinition definition,
         MethodDefinition getter = definition.declareMethod(a(PUBLIC), stateField.getGetterName(), type(stateField.getType()));
         getter.getBody()
                 .append(getter.getThis().getField(field).invoke(
-                        "get",
-                        stateField.getType(),
-                        getter.getThis().invoke("getGroupId", long.class))
+                                "get",
```

Comment:
nit: let's keep indentation consistent

Reply:
This was inserted by the IntelliJ code formatter using Presto's official codestyle XML config, so I believe it was not conforming to the code style previously? Checkstyle may be missing a rule.

Reply:
Actually, it might be a bug in the IntelliJ formatter: https://youtrack.jetbrains.com/issue/WI-50684/Continuation-indent-will-sometimes-double-indent.